src/extension/nunit.v2.driver.csproj

```diff
@@ -11,6 +11,8 @@
     <AssemblyName>nunit.v2.driver</AssemblyName>
     <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <RepositoryUrl>https://github.com/nunit-v2-framework-driver</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
```

Comment:
Since the project file isn't used to create the nuget package, this doesn't seem to serve any purpose.